broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -353,18 +355,19 @@ private void startPassiveExportingMode() {
     }
 
     if (state.hasExporters()) {
-      exporterDistributionService.subscribeForExporterPositions(actor::run);
+      exporterDistributionService.subscribeForExporterState(actor::run);
     } else {
       actor.close();
     }
   }
 
-  private void distributeExporterPositions() {
-    final var exportPositionsMessage = new ExporterPositionsMessage();
+  private void distributeExporterState() {
+    final var distributeMessage = new ExporterStateDistributeMessage();
```

Comment:
:thought_balloon: Just wondering if it would be nicer to keep with the naming pattern so it's:
```suggestion
  private void distributeExporterState() {
    final var exportStateMessage = new ExporterStateDistributeMessage();
```